pkg/operator/ceph/csi/spec.go

```diff
@@ -625,10 +625,6 @@ func validateCSIVersion(clientset kubernetes.Interface, namespace, rookImage, se
 
 	version, err := extractCephCSIVersion(stdout)
 	if err != nil {
-		if AllowUnsupported {
-			logger.Infof("failed to extract csi version, but continuing since unsupported versions are allowed. %v", err)
-			return nil
-		}
 		return errors.Wrap(err, "failed to extract ceph CSI version")
 	}
 	logger.Infof("Detected ceph CSI image version: %q", version)
```

Comment:
I wonder if it's still interesting to see this message even if unsupported...

Reply:
if `AllowUnsupported` is set to true this function wont be called

Reply:
🤦  forget about what I said.